### PR TITLE
docs(instances): ipmob

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+Another feature of IP Mobility is the ability to assign multiple flexible IP addresses to your Instance, allowing you to choose between IPv4 and IPv6 protocols.
+
+<Message type="note">
+    Currently, IP Mobility is available in the `pl-waw-2` Availability Zone.
+</Message>
+
+## How to enable IP Mobility?
+
+<Message type="note">
+  Currently, IP Mobility is disabled by default. If you're interested in exploring its functionality, you need to explicitly request it.
```
Comment (ofranc) [inline compute/instances/api-cli/using-ip-mobility.mdx:34]:
```suggestion
  Currently, IP Mobility is disabled by default. If you are interested in exploring its functionality, you need to explicitly request it.
```

Comment (ofranc) [reply compute/instances/api-cli/using-ip-mobility.mdx:34]: How to request it ?

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+</Message>
+
+In this section, we will guide you through the process of creating a new IP Mobility-enabled Instance. We will also provide instructions on migrating your existing legacy instances, keeping in mind that once the migration is complete, there is **no way to revert back**.
```
Comment (ofranc) [inline compute/instances/api-cli/using-ip-mobility.mdx:37]:
```suggestion
In this section, we will guide you through the process of creating a new IP Mobility-enabled Instance. We will also provide instructions on migrating your existing legacy Instances, keeping in mind that once the migration is complete, there is **no way to revert back**.
```

Comment (ofranc) [reply compute/instances/api-cli/using-ip-mobility.mdx:37]: Est-ce qu'on le laisse en "note" ou on change en "Important"?

**Review by ofranc — APPROVED** [review]

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 3]
```diff
@@ -0,0 +1,610 @@
...
+        ```json
+        ❯ cat payloads/migrate-data.json
+        {
+                    "action": "enable_routed_ip"
+        }
+
+        ❯ http post $API_URL/servers/$SERVER_ID/action $HEADERS <payload/migrate-data.json
+        ```
+        <Message type="note">
+            You have to reboot your Instance have to complete the migration.
```
Comment (ziirish) [inline compute/instances/api-cli/using-ip-mobility.mdx:257]:
```suggestion
            Your instance *will* reboot during this action.
```

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 4]
```diff
@@ -0,0 +1,610 @@
...
+                    "action": "enable_routed_ip"
+        }
+
+        ❯ http post $API_URL/servers/$SERVER_ID/action $HEADERS <payload/migrate-data.json
+        ```
+        <Message type="note">
+            Your instance *will* reboot during this action.
+        </Message>
+    </TabsTab>
+    <TabsTab labe="Response">
```
Comment (ldecarvalho-doc) [inline compute/instances/api-cli/using-ip-mobility.mdx:260]:
```suggestion
    <TabsTab label="Response">
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**